### PR TITLE
Style attribute HTML

### PR DESCRIPTION
**Pull request by M3RF3R**
### Description

I've  created a directory called style inside the attributes directory and inside the style directory a style.md file in which I have written a short definition for the style attribute, the syntax and an example.

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/html/concepts/attributes/style/style.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
```
Comment (yangc95) [inline content/html/concepts/attributes/style/style.md:1]: Can we create a `terms/` folder within `attributes/` and move the `style/style.md` folder and file under that terms folder? This just happens to be the structure of Docs. If you need assistance with this, please let me know and I can assist.

File: content/html/concepts/attributes/style/style.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `style` attribute specifies how an element will be rendered in a web browser, e.g. the color or the size of a font in a paragraph.
+
+When we use the `style` attribute inline, it overrides any styles set globally. 
+
+**Note:** We can also specify the style of our elements through an external CSS style sheet or embedded in the `<head>` element using the `<style>` tag.
```
Comment (yangc95) [inline content/html/concepts/attributes/style/style.md:21]:
```suggestion
> **Note:** We can also specify the style of our elements through an external CSS style sheet or embedded in the `<head>` element using the `<style>` tag.
```

Comment (yangc95) [reply content/html/concepts/attributes/style/style.md:21]: Let's refrain from using any pronouns like you and we.

File: content/html/concepts/attributes/style/style.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+## Syntax
+
+```pseudo
+<element_tag style="property1:value;property2:value;"></element_tag>
+```
+The `style` attribute supports all HTML tags and can take many properties, e.g. you can use the `style` attribute in the `<p>`, the `<h1>` or the `<hr>`.
```
Comment (yangc95) [inline content/html/concepts/attributes/style/style.md:28]:
```suggestion

The `style` attribute supports all HTML tags and can take many properties, e.g., the `style` attribute can be used in the `<p>`, the `<h1>` or the `<hr>`.
```

Comment (yangc95) [reply content/html/concepts/attributes/style/style.md:28]: I'm thinking we can set a standard here for the rest of the global attribute term entries. This is a trait that all global attributes have in that they support all HTML tags. Can we include the keyword "global attribute"

File: content/html/concepts/attributes/style/style.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+## Example
+
+In the example below, the `<p>` (paragraph) element is given a `style` attribute to specify its color, the text alignment and the font size:
```
Comment (yangc95) [inline content/html/concepts/attributes/style/style.md:32]:
```suggestion
In the example below, the `<p>` (paragraph) element is given a `style` attribute to specify its color, the text alignment, and the font size:
```

**Review by yangc95 — COMMENTED** [review]
Thanks for busting this PR out so quickly and signing the CLA @M3RF3R I conducted a review! Please tag me once you're done resolving the relevant comments or if you have any questions/need an assist.

File: content/html/concepts/attributes/style/style.md (hunk 1 above)
Comment (M3RF3R) [reply content/html/concepts/attributes/style/style.md]: Done

File: content/html/concepts/attributes/style/style.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
+---
+Title: 'Style'
+Description: 'The style attribute specifies how a page will be rendered in a web browser.'
```
[the remark this answers is not in the document]
Comment (M3RF3R) [reply content/html/concepts/attributes/style/style.md]: Done

File: content/html/concepts/attributes/style/style.md (hunk 2 above)
Comment (M3RF3R) [reply content/html/concepts/attributes/style/style.md]: Done

File: content/html/concepts/attributes/style/style.md (hunk 3 above)
Comment (M3RF3R) [reply content/html/concepts/attributes/style/style.md]: done

File: content/html/concepts/attributes/style/style.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (M3RF3R) [reply content/html/concepts/attributes/style/style.md]: I'm not sure if you want me to do anything here or just asking another collaborator?

Comment (M3RF3R) [reply content/html/concepts/attributes/style/style.md]: Done

**Comment by M3RF3R** [comment]
@yangc95 I've tried to include the changes you suggested. Thanks for he review! :)

File: content/html/concepts/attributes/terms/style/style.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `style` attribute specifies how an element will be rendered in a web browser, e.g. the color or the size of a font in a paragraph.
+
+When the `style` attribute is used inline, it overrides any styles set globally. 
+
+>**Note:** Most developers refrain from using the inline `style` attribute and set the style of the elements through an external CSS style sheet, which helps to keep the code clear and more easily maintained. The style can also be embedded in the `<head>` element using the `<style>` tag.
```
Comment (yangc95) [inline content/html/concepts/attributes/terms/style/style.md:21]:
```suggestion
> **Note:** Most developers refrain from using the inline `style` attribute and set the style of the elements through an external CSS style sheet, which helps to keep the code clear and more easily maintained. The style can also be embedded in the `<head>` element using the `<style>` tag.
```

File: content/html/concepts/attributes/terms/style/style.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+
+## Syntax
+
+```pseudo
+<element_tag style="property1:value;property2:value;"></element_tag>
+```
+Like all global attributes, the `style` attribute supports all HTML tags and can take many properties, e.g. the `style` attribute can be used in the `<p>`, the `<h1>` or the `<hr>`.
```
Comment (yangc95) [inline content/html/concepts/attributes/terms/style/style.md:28]:
```suggestion

Like all global attributes, the `style` attribute supports all HTML tags and can take many properties, e.g. the `style` attribute can be used in the `<p>`, the `<h1>` or the `<hr>`.
```

**Comment by yangc95** [comment]
@M3RF3R I left a few more comments!

File: content/html/concepts/attributes/terms/style/style.md (hunk 7 above)
Comment (M3RF3R) [reply content/html/concepts/attributes/terms/style/style.md:28]: Done

**Comment by M3RF3R** [comment]
@yangc95 I've included the suggetsed spaces. Thanks for your review

**Review by yangc95 — APPROVED** [review]
Looks great! The main change I helped make here is adding in some links: https://github.com/Codecademy/docs/pull/1539/commits/cd3df2e401a7dd5e96872bc4a2c4820bf304096f

I was quite surprised that we didn't have the `<p>` element, but it's a good thing you included it here in this entry! Whenever [the issue created](https://github.com/Codecademy/docs/issues/1566) for it is resolved, we should link it to the first mention of the term.

File: content/html/concepts/attributes/terms/style/style.md [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+  - 'Style'
+  - 'Attributes'
+  - 'Elements'
+  - 'Tags'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `style` attribute specifies how an element will render in a web browser, e.g., the color or size of a font in a paragraph.
```
Comment (caupolicandiaz) [inline content/html/concepts/attributes/terms/style/style.md:17]:
```suggestion
The **`style`** attribute specifies how an element will render in a web browser, e.g., the color or size of a font in a paragraph.
```

File: content/html/concepts/attributes/terms/style/style.md [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+```
+
+Like all global attributes, the `style` attribute supports all HTML tags and can take many properties, e.g., the `style` attribute can be used in the `<p>`, [`<h1>`](https://www.codecademy.com/resources/docs/html/elements/h1-h6), or [`<hr>`](https://www.codecademy.com/resources/docs/html/elements/hr) tags.
+
+## Example
+
+In the example below, the `<p>` element is given a `style` attribute to specify its color, the text alignment, and the font size:
+
+```html
+<p style="color:green;font-size:12px;text-align:center;">
```
Comment (caupolicandiaz) [inline content/html/concepts/attributes/terms/style/style.md:36]:
```suggestion
<p style="color:green; font-size:12px; text-align:center;">
```

File: content/html/concepts/attributes/terms/style/style.md [hunk 10]
```diff
@@ -0,0 +1,39 @@
...
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `style` attribute specifies how an element will render in a web browser, e.g., the color or size of a font in a paragraph.
+
+When the `style` attribute is used inline, it overrides any styles set globally.
+
+> **Note:** Most developers refrain from using the inline `style` attribute and set the style of the elements through an external [CSS](https://www.codecademy.com/resources/docs/css) style sheet, which helps to keep the code clear and more easily maintained. The style can also be embedded in the `<head>` element using the `<style>` tag.
```
Comment (caupolicandiaz) [inline content/html/concepts/attributes/terms/style/style.md:21]:
```suggestion
> **Note:** Most developers refrain from using the inline `style` attribute and set the style of the elements through an external [CSS](https://www.codecademy.com/resources/docs/css) style sheet, which helps to keep the code clean and more easily maintained. The styling of a page or elements on the page can also be embedded in the `<head>` element using the `<style>` tag.
```

File: content/html/concepts/attributes/terms/style/style.md [hunk 11]
```diff
@@ -0,0 +1,39 @@
...
+
+## Syntax
+
+```pseudo
+<element_tag style="property1:value;property2:value;"></element_tag>
```
Comment (caupolicandiaz) [inline content/html/concepts/attributes/terms/style/style.md:26]:
```suggestion
<element_tag style="property1:value; property2:value;"></element_tag>
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@M3RF3R Just a few small changes

File: content/html/concepts/attributes/terms/style/style.md (hunk 8 above)
Comment (M3RF3R) [reply content/html/concepts/attributes/terms/style/style.md:17]: done

File: content/html/concepts/attributes/terms/style/style.md (hunk 9 above)
Comment (M3RF3R) [reply content/html/concepts/attributes/terms/style/style.md:36]: Done

File: content/html/concepts/attributes/terms/style/style.md (hunk 10 above)
Comment (M3RF3R) [reply content/html/concepts/attributes/terms/style/style.md:21]: Done

File: content/html/concepts/attributes/terms/style/style.md (hunk 11 above)
Comment (M3RF3R) [reply content/html/concepts/attributes/terms/style/style.md:26]: Done

**Comment by M3RF3R** [comment]
@caupolicandiaz Thanks for your review! I've added your changes

File: content/html/concepts/attributes/terms/style/style.md [hunk 12]
```diff
@@ -0,0 +1,39 @@
...
+  - 'Elements'
+  - 'Tags'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The **`style`** attribute specifies how an element will render in a web browser, e.g., the color or size of a font in a paragraph.
+
+When the **`style`** attribute is used inline, it overrides any styles set globally.
```
Comment (caupolicandiaz) [inline content/html/concepts/attributes/terms/style/style.md:19]:
```suggestion
When the `style` attribute is used inline, it overrides any styles set globally.
```

File: content/html/concepts/attributes/terms/style/style.md [hunk 13]
```diff
@@ -0,0 +1,39 @@
...
+
+> **Note:** Most developers refrain from using the inline **`style`** attribute and set the style of the elements through an external [CSS](https://www.codecademy.com/resources/docs/css) style sheet, which helps to keep the code clean and more easily maintained. The styling of a page or elements on the page can also be embedded in the `<head>` element using the `<style>` tag.
+
+## Syntax
+
+```pseudo
+<element_tag style="property1:value; property2:value;"></element_tag>
+```
+
+Like all global attributes, the **`style`** attribute supports all HTML tags and can take many properties, e.g., the **`style`**git attribute can be used in the `<p>`, [`<h1>`](https://www.codecademy.com/resources/docs/html/elements/h1-h6), or [`<hr>`](https://www.codecademy.com/resources/docs/html/elements/hr) tags.
```
Comment (caupolicandiaz) [inline content/html/concepts/attributes/terms/style/style.md:29]:
```suggestion
Like all global attributes, the `style` attribute supports all HTML tags and can take many properties, e.g., the `style` attribute can be used in the `<p>`, [`<h1>`](https://www.codecademy.com/resources/docs/html/elements/h1-h6), or [`<hr>`](https://www.codecademy.com/resources/docs/html/elements/hr) tags.
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@M3RF3R Overall, this looks good. But, we only want the term in bold in the first instance.

**Comment by M3RF3R** [comment]
@caupolicandiaz ok sorry! think I've done it!

**Review by caupolicandiaz — APPROVED** [review]
@M3RF3R All set!

**Comment by M3RF3R** [comment]
@caupolicandiaz great! Is there anything else that I need to do? Can I go ahead and delete the style branch I creted in my repository?

**Comment by M3RF3R** [comment]
or click the close pull request button here?

**Comment by M3RF3R** [comment]
@caupolicandiaz Hi! I'm wondering if there is a problem with this entry. What hasn't it been merged?

**Comment by yangc95** [comment]
> @caupolicandiaz Hi! I'm wondering if there is a problem with this entry. What hasn't it been merged?

Thanks for your patience with getting this entry ready for merge and up to standards! There's no issue with this entry that makes it ineligible for merge.

File: content/html/concepts/attributes/terms/style/style.md [hunk 14]
```diff
@@ -0,0 +1,39 @@
...
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The **`style`** attribute specifies how an element will render in a web browser, e.g., the color or size of a font in a paragraph.
+
+When the `style` attribute is used inline, it overrides any styles set globally.
+
+> **Note:** Most developers refrain from using the inline **`style`** attribute and set the style of the elements through an external [CSS](https://www.codecademy.com/resources/docs/css) style sheet, which helps to keep the code clean and more easily maintained. The styling of a page or elements on the page can also be embedded in the `<head>` element using the `<style>` tag.
```
Comment (yangc95) [inline content/html/concepts/attributes/terms/style/style.md:21]:
```suggestion
> **Note:** The styling of a page or elements on the page can also be embedded in the `<head>` element using the [`<style>`](https://www.codecademy.com/resources/docs/html/elements/style) tag. However, most developers refrain from using the inline `style` attribute and set the style of the elements through an external [CSS](https://www.codecademy.com/resources/docs/css) style sheet. This helps to keep the code clean and is more easily maintained.
```

**Comment by M3RF3R** [comment]
@yangc95 Thanks for your quick reply! I've commited your suggestions.

**Review by yangc95 — APPROVED** [review]

**Comment by yangc95** [comment]
@M3RF3R here's the live link: https://www.codecademy.com/resources/docs/html/attributes/style

Another one in the bag!
Screen Shot 2023-02-26 at 10 33 29 PM

cc: @caupolicandiaz @KTom101

**Pull request merged**